Extension/src/LanguageServer/configurations.ts

```diff
@@ -1241,6 +1238,17 @@ export class CppProperties {
             this.resetToDefaultSettings(true);  // I don't think there's a case where this will be hit anymore.
         }
 
+        const configuration: Configuration | undefined = this.CurrentConfiguration;
+        if (configuration) {
+            if (configuration.configurationProvider !== undefined) {
+                getUI().then((ui: LanguageStatusUI) => ui.ShowConfigureIntelliSenseButton(false, this.client, ConfigurationType.ConfigProvider, "baseConfiguration"));
```

Comment:
This appears to be causing the button to disappear after a configuration change or am I mistaken?